google-cloud-examples/src/test/java/com/google/cloud/examples/storage/snippets/ITStorageSnippets.java

```diff
@@ -383,22 +393,22 @@ public void testBlobAcl() {
     assertNull(storageSnippets.getBlobAcl(BUCKET, blobName, createdBlob.getGeneration()));
     // test non-existing blob
     String nonExistingBlob = "test-blob-acl";
-    assertNull(storageSnippets.getBlobAcl(BUCKET, nonExistingBlob, -1L));
-    assertFalse(storageSnippets.deleteBlobAcl(BUCKET, nonExistingBlob, -1L));
+    assertNull(storageSnippets.getBlobAcl(BUCKET, nonExistingBlob, 1L));
```

Comment:
I apologize for not knowing this very well. What does the last parameter mean and why is it changing?

Reply:
The test was failing because of a 400 response from the server. I'm guessing that there was a change on the API's backend that no longer allows for negative numbers? 

@frankyn Can you provide some more info?

Reply:
At some point, -1L seemed to be allowed, but is no longer allowed and causes an exception. The change should resolve the exception and respond with a NotFound instead.  Not sure when this change hit production though.

Reply:
Makes sense to me.